### PR TITLE
Remove manifest_v2 flag

### PR DESCRIPTION
**Pull request by teresaromero**
## Proposed changes

- remove references to MANIFEST_V2 flag
- running buildv1 or buildv2 depending on manifest version and user input

### CODE REVIEW

File: cmd/deploy/deploy.go [hunk 1]
```diff
@@ -214,53 +214,48 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
...
-		if deployOptions.Manifest.Build != nil {
-
-			for service, mOptions := range deployOptions.Manifest.Build {
-				buildOptions := build.BuildOptions{OutputMode: oktetoLog.GetOutputFormat()}
-				if mOptions.Name == "" {
-					mOptions.Name = deployOptions.Name
-				}
-				opts := build.OptsFromManifest(service, mOptions, buildOptions)
+	if deployOptions.Dependencies && !okteto.IsOkteto() {
+		return fmt.Errorf("deploy of dependencies is only available for Okteto instances")
```
Comment (pchico83) [inline cmd/deploy/deploy.go:218]:
```suggestion
		return fmt.Errorf("deploy of dependencies is only available in clusters managed by Okteto")
```

**Review by pchico83 — APPROVED** [review]

File: cmd/build.go [hunk 2]
```diff
@@ -38,15 +39,48 @@ func Build(ctx context.Context) *cobra.Command {
 	options := build.BuildOptions{}
 	cmd := &cobra.Command{
 		Use:   "build [PATH]",
-		Args:  utils.BuildMaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#build"),
 		Short: "Build (and optionally push) a Docker image",
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{}); err != nil {
```
Comment (jLopezbarb) [inline cmd/build.go:45]: I think we need to run this to setup secrets and other information

Comment (jLopezbarb) [reply cmd/build.go:45]: Nevermid I just saw its removing duplicated code

**Review by jLopezbarb — APPROVED** [review]

File: cmd/build.go (hunk 2 above)
Comment (teresaromero) [reply cmd/build.go:45]: secrets is not supported as build param at the manifest, the secrets are allowed as flag if single build is done.

**Pull request merged**